hooks/pre/34-pulpcore_directory_layout.rb

```diff
@@ -1,7 +1,7 @@
 require 'pathname'
 
 PULP_ROOT = Pathname.new('/var/lib/pulp')
-LEGACY_DIR = PULP_ROOT / 'artifact'
+LEGACY_DIR = PULP_ROOT / 'docroot'
```

Comment:
This may actually be `/var/lib/pulp/docroot/artifact`. In that case, it may make more sense to rename `docroot` to `media`. It also breaks `DESTINATION`.